posix.mak

```diff
@@ -283,7 +283,7 @@ install :
 
 .PHONY : unittest
 ifeq (1,$(BUILD_WAS_SPECIFIED))
-unittest : $(addsuffix .d,$(addprefix unittest/,$(D_MODULES)))
+unittest : $(addsuffix .run,$(addprefix unittest/,$(D_MODULES)))
```

Comment:
You gotta love make, if I mark them `.PHONY`, make stops executing implicit targets for them.
http://stackoverflow.com/questions/3095569/when-declaring-the-pattern-rule-as-phony-it-is-not-triggered/3095711#answer-3095711

Let's not do this, the output files are never actually created, so the rules are already rerun without `.PHONY`.